packages/plugin-core/src/constants.ts

```diff
@@ -358,6 +359,38 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     docPreview: "",
     when: DendronContext.PLUGIN_ACTIVE,
   },
+
+  // This command will only apply when the note look up quick pick is open
+  // which is taken care by the DendronContext.NOTE_LOOK_UP_ACTIVE
+  //
+  // It will also NOT activate when the focus is in editor using `!editorFocus`
+  //
+  // However, when it comes to user navigating to side panels its quite imperfect.
+  // We do have some protection against Tab interception by using the `!view`
+  // (most side panels set the view variable Eg. "view": "dendron.backlinks").
+  // But it is possible for user to tab into empty side panel which does not
+  // have a `view` context set, at that point if user still has look up open and
+  // presses tab, Tab will get intercepted by note auto complete.
+  //
+  // Ideally there would be a trigger event when quick pick goes in focus/focuses out
+  // but not able to find such hook.
+  LOOKUP_NOTE_AUTO_COMPLETE: {
+    key: "dendron.lookupNoteAutoComplete",
+
+    // TODO: Hide this command from command palette
```

Comment:
Can we try this? https://stackoverflow.com/questions/55270915/vs-code-extension-hide-commands-from-command-palette

`package.json`:
```typescript
"contributes": {
"commands": [
    {
        "command": "extension.myHiddenCommand",
        "title": "Compile folder"
    }
],
"menus": {
    "commandPalette": [
        {
            "command": "extension.myHiddenCommand",
            "when": "false"
        }
    ]
}
```

Reply:
Will try

Reply:
Works